docs/config.md

```diff
@@ -0,0 +1,59 @@
+## Configuring Google Spreadsheet Operations
+
+To use the Google Spreadsheet connector, add the `<googlespreadsheet.init>` element in your proxy configuration before use any other
+Google Spreadsheet connector operations. This `<googlespreadsheet.init>` element used to authenticates  the user using OAuth2 authentication
+to access the Google account which contains the spreadsheets. For more information on authorizing requests in Google Spreadsheets, see [API Doc](https://developers.google.com/sheets/api/guides/authorizing).
+
+<br/>
+
+> Follow this [guide](https://docs.wso2.com/display/IntegrationCloud/Get+Credentials+for+Google+Spreadsheet) to set up the Google Spreadsheets and get credentials such as clientId, clientSecret, accessToken, refreshToken.
```

Comment:
Follow this [guide](https://docs.wso2.com/display/IntegrationCloud/Get+Credentials+for+Google+Spreadsheet) to set up the Google Spreadsheets and get credentials such as the clientId, clientSecret, accessToken, and the refreshToken.